src/java/org/apache/cassandra/utils/ExpiringMemoizingSupplier.java

```diff
@@ -77,6 +77,11 @@ public T get() {
         return this.value;
     }
 
+    public void expire()
```

Comment:
nit: annotate with `@VisibleForTesting`? It is only used for testing.

Reply:
ok